docker-compose.yaml

```diff
@@ -1,20 +1,42 @@
-version: '3'
+version: '3.7'
 services:
-  # ceqr service
-  ceqr:
+  # ceqr app service
+  ceqr_service:
     build: .
-    entrypoint: ./entrypoint.sh
+    entrypoint: ./service-entrypoint.sh
     volumes:
       - .:/app
     ports:
       - "3000:3000"
     environment:
       PORT: 3000
       DATABASE_URL: postgis://postgres@postgis:5432/ceqr_rails
-      CEQR_DATA_DB_URL: postgis://test_ceqr_app:rhqnwjwiup4yafzw@cear-data-do-user-1939427-0.db.ondigitalocean.com:25060/ceqr_data
+      CEQR_DATA_DB_URL: postgis://test_ceqr_app:${PSQL_PASSWORD}@ceqr-data-readonly-do-user-1939427-0.db.ondigitalocean.com:25060/ceqr_data
```

Comment:
Awesome! We def need to keep the read-only credentials private, too.

Reply:
I re-set the password for this user, so the one that was previously in version control is useless